tests/test_formatters.py

```diff
@@ -206,6 +207,47 @@ def test_escaping_govuk_in_email_templates(template_content, expected):
     assert expected in str(HTMLEmailTemplate(template_json))
 
 
+@pytest.mark.parametrize(
+    "template_content,expected", [
+        # Cases that we add the breaking space
+        ("GOV.UK", u"GOV.\u200BUK"),
+        ("gov.uk", u"gov.\u200Buk"),
+        ("content with space infront GOV.UK", u"content with space infront GOV.\u200BUK"),
+        ("content with tab infront\tGOV.UK", u"content with tab infront\tGOV.\u200BUK"),
+        ("content with newline infront\nGOV.UK", u"content with newline infront\nGOV.\u200BUK"),
+        ("*GOV.UK", u"*GOV.\u200BUK"),
+        ("#GOV.UK", u"#GOV.\u200BUK"),
+        ("^GOV.UK", u"^GOV.\u200BUK"),
+
+        # Cases that we don't add the breaking space
+        ("https://gov.uk", "https://gov.uk"),
+        ("https://www.gov.uk", "https://www.gov.uk"),
+        ("www.gov.uk", "www.gov.uk"),
+        ("WWW.GOV.UK", "WWW.GOV.UK"),
+        ("https://www.gov.uk/?utm_source=gov.uk", "https://www.gov.uk/?utm_source=gov.uk"),
+        ("www.this-site-is-not-gov.uk", "www.this-site-is-not-gov.uk"),
+        ("www.gov.uk?websites=bbc.co.uk;gov.uk;nsh.scot", "www.gov.uk?websites=bbc.co.uk;gov.uk;nsh.scot"),
+        ("reply to: xxxx@xxx.gov.uk", "reply to: xxxx@xxx.gov.uk"),
+        ("coronavirus.data.gov.uk", "coronavirus.data.gov.uk"),
+        ("gov.uk/foo", "gov.uk/foo"),
+        ("*GOV.UK/foo", "*GOV.UK/foo"),
+        ("#GOV.UK/foo", "#GOV.UK/foo"),
+        ("^GOV.UK/foo", "^GOV.UK/foo"),
+
+        # Cases I'm not sure on
+        ("data.gov.uk", "data.gov.uk"), # depends if this is a special case like gov.uk is a brand
+        ("southwark.gov.uk", "southwark.gov.uk"), # i lean towards making this a link
+        ("#gov.uk/register-to-vote", "#gov.uk/register-to-vote") # i lean towards making this a link
```

Comment:
we currently make all three of them links (based on the leading `.` for the first two, and the trailing `/` for the last one)

i think we keep that behaviour. Deffo last two I agree with you. data.gov.uk's a bit more interesting but I dont know enough about branding etc on that.